src/sentry/workflow_engine/models/detector_group.py

```diff
@@ -13,12 +13,11 @@ class DetectorGroup(DefaultFieldsModel):
     __relocation_scope__ = RelocationScope.Excluded
 
     detector = FlexibleForeignKey("workflow_engine.Detector", on_delete=models.CASCADE)
-    group = FlexibleForeignKey("sentry.Group", on_delete=models.CASCADE)
+    group = FlexibleForeignKey("sentry.Group", on_delete=models.CASCADE, unique=True)
 
     class Meta:
         db_table = "workflow_engine_detectorgroup"
         app_label = "workflow_engine"
-        unique_together = (("detector", "group"),)
```

Comment:
I think we're safe enough without this index. If we search on `detector_id, group_id` it can just use the unique on group. The only case I can think of where this old index might have been helpful if we did `where detector_id = x order by group_id`, which doesn't seem like it'd happen too often